app/Jobs/ProcessInvoicePaid.php

```
@@ -0,0 +1,54 @@
+    public array $payload;
+
+    /** @param array<string, mixed> $payload */
+    public function __construct(array $payload)
+    {
+        $this->payload = $payload;
+    }
+
+    public function handle(): void
+    {
+        $invoice = (array) Arr::get($this->payload, 'data.object', []);
+        $lines = (array) ($invoice['lines']['data'] ?? []);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$lines'. (undefined)

(UnusedLocalVariable)